### DISCUSSION TITLE
Missing combined_reference_data_grch37.vds on cloud

### DISCUSSION THREAD

**Discussion by samarthaFFM** [broadinstitute/seqr#2578] [Q&A]
Hi Seqr team, 

In the past few days, I am unable to upload vcf files through hail on to my local installation of SEQR because of the following error - 

Java stack trace:
is.hail.utils.HailException: arguments refer to no files: 'gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.vds'
        at is.hail.utils.ErrorHandling$class.fatal(ErrorHandling.scala:6)
        at is.hail.utils.package$.fatal(package.scala:27)
        at is.hail.HailContext.readAll(HailContext.scala:431)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:237)
        at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
        at py4j.Gateway.invoke(Gateway.java:280)
        at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
        at py4j.commands.CallCommand.execute(CallCommand.java:79)
        at py4j.GatewayConnection.run(GatewayConnection.java:214)
        at java.lang.Thread.run(Thread.java:748)

Hail version: 0.1-105a497
Error summary: HailException: arguments refer to no files: 'gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.vds'

Could you please help? 

Thanks, 
Samarth

**Comment by hanars** [comment]
Can you please provide details of the command you are running? Using the latest docker image and the latest recommended commands for loading, I would not expect you to be referencing "gs://seqr-reference-data/GRCh37/all_reference_data/combined_reference_data_grch37.vds'" at all. Latest instructions are here: https://github.com/broadinstitute/seqr/blob/master/deploy/LOCAL_INSTALL.md#annotating-and-loading-vcf-callsets

**Reply by samarthaFFM** [reply]
Hi @hanars, 

I was not able to install the docker image because of errors at my local server and this is a version that was released just before the docker image and submits the following command - 


/home/ttysam01/seqr/../bin/spark-2.0.2-bin-hadoop2.7/bin/spark-submit                  --driver-memory 125g         --executor-memory 100g         --num-executors 150         --conf spark.driver.extraJavaOptions=-Xss4M         --conf spark.executor.extraJavaOptions=-Xss4M         --conf spark.executor.memoryOverhead=20g         --conf spark.driver.maxResultSize=35g         --conf spark.kryoserializer.buffer.max=1g         --conf spark.memory.fraction=0.1         --conf spark.default.parallelism=1         --conf spark.executor.heartbeatInterval=10000000         --conf spark.network.timeout=10000001         --jars hail_builds/v01/hail-v01-10-8-2018-90c855449.jar         --conf spark.driver.extraClassPath=hail_builds/v01/hail-v01-10-8-2018-90c855449.jar         --conf spark.executor.extraClassPath=hail_builds/v01/hail-v01-10-8-2018-90c855449.jar         --py-files hail_builds/v01/hail-v01-10-8-2018-90c855449.zip         "hail_scripts/v01/load_dataset_to_es.py" "--genome-version" "37" "--project-guid" "Project_FAM127" "--sample-type" "WES" "--dataset-type" "VARIANTS" "--skip-validation" "--exclude-hgmd" "--vep-block-size" "100" "--es-block-size" "150" "--num-shards" "5" "--use-nested-objects-for-vep" "--use-nested-objects-for-genotypes" "--index" "FAM127_01" "--host" "192.168.3.40" "--port" "9203" "/is3/PUBLIC/seqr/families_2//./FAM127/all.vcf" --username 'root' --directory 'vls188-seqr.bioinfo:/home/ttysam01/Seqr_install/seqr/hail_elasticsearch_pipelines'


Basically, it was in service till yesterday after which it started failing. Could you please help?

**Reply by samarthaFFM** [reply]
Is it still possible to offer legacy support for this version so that I could access the reference files? Not sure if it changed in the last few days.

**Reply by hanars** [reply]
I'm sorry but we do not provide support for deprecated versions of code, the only supported mode for running the seqr pipeline locally is via docker image. The fact that we happened to not cause a breaking change for years was a nice coincidence, not a supported use case. The code you are using is so out of date I can't even figure out how to help you reconfigure the command to work. I recommend you instead try to debug the issues with installing the docker image

**Reply by hanars** [reply]
them outdated reference files were deleted because they are supposed to have been unused for years and we can't really afford to continue to pay for them indefinitely

**Reply by samarthaFFM** [reply]
Thank you for the answers @hanars. We used to update the SEQR sql databases as suggested and kept operating them without changing the SEQR version since it worked well for us. I will try debugging the SEQR docker image and work on that for the new files. Thanks!

**Reply by hanars** [reply]
If you are not currently using the seqr docker images at all, you shouldn't need to update the rest of the seqr to use the docker images, you can just use the pipeline runner one on its own. Use it as a standalone docker image instead of as part of the docker-compose file.

That said, we do recommend eventually updating as there are a lot of known security vulnerabilities in old versions of seqr
